code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -301,12 +303,53 @@
 					if(prob(40))
 						M.emote("scream")
 					M.mind.vampire.nullified = max(5, M.mind.vampire.nullified + 2)
+	else if(ishuman(M) && M.mind && M.mind.vampire && M.mind.vampire.get_ability(/datum/vampire_passive/full) && prob(80)) //Dirty copypaste for vampire fullpower, 500 blood to infinity, they take more damage from holy water
```

Comment:
:angry: `//Dirty copypaste`. I suggest making this section and the one above it, into a proc, with the ability to tweak the severity of the jitter, stutter etc.